README.md

```diff
@@ -1,11 +1,16 @@
 # Geocortex Mobile Quickstart
+
+![CI](https://github.com/geocortex/vertigis-mobile-quickstart/workflows/CI/badge.svg)
+
 A Geocortex Mobile app template to facilitate getting started with the Geocortex Mobile SDK and custom development.
 
 ## The Quickstart
-- Open the **App1** solution in Visual Studio.
-- Set **App1.UWP** as your startup project.
-- Change the architecture from _Any CPU_ to _x86_ or _x64_.
-- Build and run.
+
+-   Open the **App1** solution in Visual Studio.
+-   Set **App1.UWP** as your startup project.
+-   Change the architecture from _Any CPU_ to _x86_ or _x64_.
+-   Build and run.
 
 ## Documentation
-- Find [further documentation on the quickstart](https://developers.geocortex.com/docs/mobile/quick-start/) on the [Geocortex Development Center](https://developers.geocortex.com/docs/mobile/overview).
+
+-   Find [further documentation on the quickstart](https://developers.geocortex.com/docs/mobile/quick-start/) on the [Geocortex Development Center](https://developers.geocortex.com/docs/mobile/overview).
```

Comment:
The link to the quick-start isn't valid anymore. I think you'll want this one:
https://developers.geocortex.com/docs/mobile/sdk-quick-start

Reply:
Good catch! I fixed in #3.